local-interchaintest/src/tests/remote_chain_splitter/remote_chain_splitter.rs

```diff
@@ -0,0 +1,307 @@
+use crate::helpers::constants::{
+    ACC1_ADDRESS_NEUTRON, ACC2_ADDRESS_NEUTRON, ASTROPORT_PATH, LOCAL_CODE_ID_CACHE_PATH,
+    VALENCE_PATH,
+};
+use cosmwasm_std::{Decimal, Uint128, Uint64};
+use covenant_utils::{op_mode::ContractOperationModeConfig, split::SplitConfig};
+use localic_std::{errors::LocalError, modules::cosmwasm::CosmWasm};
+use localic_utils::{
+    utils::test_context::TestContext, DEFAULT_KEY, GAIA_CHAIN_NAME, NEUTRON_CHAIN_ID,
+    NEUTRON_CHAIN_NAME,
+};
+use serde_json::Value;
+use std::{
+    collections::BTreeMap,
+    time::{Duration, Instant},
+};
+use valence_covenant_single_party_pol::msg::DEFAULT_TIMEOUT;
+
+use log::info;
+
+struct ContractInfo {
```

Comment:
Why you need this? I think you can reuse the Contract struct in localic-std. For the execute of the Cosmwasm object that you are creating later you don't need the code_id so it would be fine